net/redis/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2007-2015 OpenWrt.org
```

Comment:
IANAL, but in my jurisdiction there is implied copyright on any produced works.  Not sure what's involved in transfer of copyright, but probably muddied by differences in international law.  In any case, as mentioned to @artynet, I'm happy for you to use the work, this stuff is complicated though.

I would also be happy to apply a license to the repo now that is compatible for merging stuff here if that would help.